infermedica_api/models/diagnosis.py

```diff
@@ -117,7 +117,7 @@ def __init__(self, sex, age, time=None, interview_id=None, **kwargs):
     def observations(self):
         return self.symptoms
 
-    def __add_evidence(self, collection, _id, state, time, initial):
+    def __add_evidence(self, collection, _id, state, time, source):
```

Comment:
Hmm the question is now if we want to do a breaking change or mark the initial argument as deprecated and leave as a legacy option. "source" will be added next to it, and in some feature releases we will remove the initial argument. But that will leave more time to switch to new solution for users.

Reply:
Yes, It would be an option as well. If other guys agree, then I'll change it in suggested way.

Reply:
I'd opt for backward compatibility